torch/nn/modules/transformer.py

```diff
@@ -251,6 +252,8 @@ def __init__(self, d_model, nhead, dim_feedforward=2048, dropout=0.1):
         self.dropout1 = Dropout(dropout)
         self.dropout2 = Dropout(dropout)
 
+        self.activation = F.relu if activation == "relu" else F.gelu
```

Comment:
Should this error out instead if something other than relu/gelu was passed in?

Reply:
good point